src/js/routes/TwitterHandleLanding.jsx

```diff
@@ -41,10 +41,10 @@ export default class TwitterHandleLanding extends Component {
 
   componentWillReceiveProps (nextProps) {
     // console.log("TwitterHandleLanding componentWillReceiveProps");
-    if (nextProps.params.twitter_handle && this.state.twitter_handle.toLowerCase() !== nextProps.params.twitter_handle.toLowerCase()) {
+    if (nextProps.params.twitterHandle && this.state.twitterHandle.toLowerCase() !== nextProps.params.twitterHandle.toLowerCase()) {
```

Comment:
This should be reverted to `nextProps.params.twitter_handle` per my earlier comment.